content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -92,7 +92,7 @@ export class Hero extends React.PureComponent {
               </div>
             </div>
             <ImpressionStats
-              campaignId={heroRec.campaignId}
+              campaignId={heroRec.campaign_id}
```

Comment:
Good catch, it should be same as <DSCard campaignId={rec.campaign_id} above which is recording impressions correct.